custom_domains/getanid/main.tf

```diff
@@ -0,0 +1,33 @@
+module "domains" {
+  source              = "git::https://github.com/DFE-Digital/terraform-modules.git//domains/environment_domains"
+  zone                = var.zone
+  front_door_name     = var.front_door_name
+  resource_group_name = var.resource_group_name
+  domains             = var.domains
+  tags                = local.tags
+  environment         = var.environment_short
+  service_name        = var.service_name
+  host_name           = data.azurerm_linux_web_app.app.default_hostname
+
+}
+
+data "azurerm_cdn_frontdoor_profile" "main" {
+  name                = var.front_door_name
+  resource_group_name = var.resource_group_name
+}
+
+data "azurerm_linux_web_app" "app" {
+  provider            = azurerm.transversal
+  name                = var.app_name
+  resource_group_name = var.app_resource_group_name
+}
+
+data "azurerm_dns_zone" "main" {
+  name                = var.zone
+  resource_group_name = var.resource_group_name
+}
+
+resource "azurerm_cdn_frontdoor_rule_set" "ruleset" {
```

Comment:
Can we configure the redirect rule using terraform as well? With: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_frontdoor_rule